docs/modules/write-the-docs/pages/glossary.adoc

```diff
@@ -0,0 +1,653 @@
+
+[[glossary-mouse]] mouse:: The device that the user moves on their desk to control the <<glossary-cursor, cursor>>.
++
+In general, do not refer to the mouse or the cursor.
+Instead, tell the user what to do and assume they know how to do it.
++
+[options="header", cols="2,2"]
+|===
+| Incorrect
+| Correct
+
+| Click the right button on the mouse and select *Print*.
+| Right-click and select *Print*.
+
+| Use the pointer to click *Submit*.
+| Click *Submit*.
+|===
+
+== N
+
+need:: Use instead of "must have" or "necessary."
++
+Use "if needed" or "as needed" rather than "if you need to."
+
+never:: Avoid unless it is an accurate description.
++
+See <<glossary-all-always, all and always>>.
+
+[[glossary-null]] null:: Use to describe a database field that is `null`.
+If the database field is empty, use <<glossary-empty, empty>> instead, not "blank."
+
+== O
+
+on-site versus on site:: Hyphenate "on-site" when using the phrase as a compound adjective.
+With verbs, use "on site" (two words).
++
+.Use of "on-site" and "on site" in sentences
+====
+There will be an on-site meeting.
+
+Training is provided on site.
+====
+
+only:: "Only" is a modifier, and should be used as such.
+
+[[glossary-option]] option:: Use instead of <<glossary-radio-button, radio button>>, unless describing the element type in developer content.
+Use <<glossary-select, select>> and <<glossary-clear, clear>> to describe interacting with an option.
++
+[options="header", cols="2,2"]
+|===
+| Incorrect
+| Correct
+
+| Click the *I Agree* radio button.
+| Click the *I Agree* option.
+|===
+
+== P
+
+platform:: A generic term for the back-end hardware, software, and services that support the operation of a portal.
+
+pre-:: A prefix meaning "before."
+This prefix is usually affixed to words without using a hyphen (for example, "prefix," "predate").
+Use a hyphen in the following cases:
++
+* Where excluding the hyphen might lead to a mispronunciation or misunderstanding of the word (for example, "pre-judicial," meaning "before being a judge," could be read as "prejudicial," which has a very different meaning).
+* Before a character other than a letter (for example, "pre-1960").
+* Before a proper noun (for example, "pre-World War I").
+
+preset:: Initial, default settings configured in a freshly installed program.
++
+Compare to <<glossary-default, default>>.
+
+press:: Verb to use when referring to a keyboard key.
+Use instead of "depress," "strike," or "hit."
++
+[options="header", cols="2,2"]
+|===
+| Incorrect
+| Correct
+
+| Hit Enter to continue.
+| Press Enter to continue.
+|===
+
+product, project:: The official terms for our software.
+Do not use "application," "component," or "program."
+
+prompts and labels:: Do not include the punctuation from a prompt or field label on the UI, or from the default entry associated with the prompt or field label.
+An exception to this rule is when the prompt or field label ends in a question mark.
```

Comment:
The example names the field without the question mark.

Reply:
Oh, true. I might have reversed them somehow. I'll fix that up!